### DISCUSSION TITLE
反复要求验证身份

### DISCUSSION THREAD

**Discussion by penndu** [ArtalkJS/Artalk#151] [Q&A] [answered]
如图：

![Honeycam 2022-05-15 23-01-41](https://user-images.githubusercontent.com/46226528/168753481-a0902d2d-190f-4e07-bc5f-dbd08f7fd89c.gif)

**Comment by qwqcode** [comment, marked as answer]
好奇怪，刷新一下 CDN 缓存试试？

**Reply by penndu** [reply]
清了，还是不行！

**Reply by penndu** [reply]
可能是插件的问题，卸载之后就可以了！
